autoload/clap/provider/buffers.vim

```diff
@@ -34,13 +34,16 @@ function! s:buffers() abort
     silent buffers
   redir END
   let s:line_info = {}
-  for line in split(l:buffers, "\n")
-    let bufnr = str2nr(trim(matchstr(line, '^\s*\d\+')))
-    let lnum = matchstr(line, '\s\+\zsline.*$')
-    let s:line_info[bufnr] = lnum
-  endfor
-  let bufs = map(clap#util#buflisted_sorted(), 's:format_buffer(str2nr(v:val))')
-  return bufs[1:] + [bufs[0]]
+  if len(l:buffers) > 0
+    for line in split(l:buffers, "\n")
+      let bufnr = str2nr(trim(matchstr(line, '^\s*\d\+')))
+      let lnum = matchstr(line, '\s\+\zsline.*$')
+      let s:line_info[bufnr] = lnum
+    endfor
+    let bufs = map(clap#util#buflisted_sorted(), 's:format_buffer(str2nr(v:val))')
+    return bufs[1:] + [bufs[0]]
```

Comment:
It could be safer to change to this as `clap#util#buflisted_sorted()` filters the qf out by default :

```vim
if empty(bufs)
  return []
else
  return bufs[1:] + [bufs[0]]
endif
```

Reply:
Resolved in 324dac2